app/scripts/controllers/loanAccount/NewJLGLoanAccAppController.js

```diff
@@ -14,7 +14,7 @@
             scope.loanApplicationCommonData.submittedOnDate = new Date();
             scope.loanApplicationCommonData.expectedDisbursementDate = new Date();
             scope.loanApplicationCommonData.syncDisbursementWithMeeting = true;
-
+			scope.errorStatus="The next meeting date must be defined";
```

Comment:
how do I call the text in the controller once it has been moved to the
locale.json

On Tue, Feb 23, 2016 at 5:01 AM, Shaik Nazeer Hussain <
notifications@github.com> wrote:

> In app/scripts/controllers/loanAccount/NewJLGLoanAccAppController.js
> <https://github.com/openMF/community-app/pull/1477#discussion_r53758525>:
>
> > @@ -14,7 +14,7 @@
> >              scope.loanApplicationCommonData.submittedOnDate = new Date();
> >              scope.loanApplicationCommonData.expectedDisbursementDate = new Date();
> >              scope.loanApplicationCommonData.syncDisbursementWithMeeting = true;
> > -
> > +			scope.errorStatus="The next meeting date must be defined";
>
> Please move text to locale.json
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openMF/community-app/pull/1477/files#r53758525>.
>